core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/Purchases.kt

```diff
@@ -93,12 +91,6 @@ public expect class Purchases {
         )
     }
 
-    /**
-     * Default to [REVENUECAT], set this to [MY_APP] if you are consuming and acknowledging
-     * transactions outside of the Purchases SDK.
-     */
-    public var purchasesAreCompletedBy: PurchasesAreCompletedBy
```

Comment:
This is so `purchasesAreCompletedBy` cannot be changed after configuration, right? If so 👍

Reply:
Yeah! I couldn't think of a good use case to change it, so we decided to remove it.